### DISCUSSION TITLE
The possibility to create or find new cheats and add to my personal local cheats?

### DISCUSSION THREAD

**Discussion by ScottSummers616** [bucanero/ArtemisPS3#89] [General]
Hello guys i am new to this so i have a few questions, is there a way to find cheats that may not be on the database yet for a specific game and add it?
 I am having a hard time with Budokai HD Collection (Budokai 3 to be exact) where i want a infinite HP cheat but there is none for some reason, how can i find it, add it?

**Comment by bucanero** [comment]
You'll need to have CFW installed and use NetCheat with your computer to find new codes.

For details check this link: https://github.com/bucanero/ArtemisPS3/issues/81#issuecomment-2470456271

**Reply by ScottSummers616** [reply]
thank you for the response, i am glad to know there is a way, i will take a look, i am VERY curious how all this works
